src/odbc_statement.cpp

```diff
@@ -48,6 +48,9 @@ ODBCStatement::ODBCStatement(const Napi::CallbackInfo& info) : Napi::ObjectWrap<
   this->data = new StatementData();
   this->odbcConnection = info[0].As<Napi::External<ODBCConnection>>().Data();
   this->data->hstmt = *(info[1].As<Napi::External<SQLHSTMT>>().Data());
+  this->data->fetch_array         = this->odbcConnection->connectionOptions.fetchArray;
+  this->data->maxColumnNameLength = this->odbcConnection->getInfoResults.max_column_name_length;
+  this->data->get_data_supports   = this->odbcConnection->getInfoResults.sql_get_data_supports;
```

Comment:
This works, but it might make sense to add a StatementData constructor. That would ensure these are always set when StatementData is created. Depends on how often these are constructed though, if it's just here or one other spot, it's probably overkill.